queries.sql

```diff
@@ -98,4 +98,14 @@ FROM animals
 WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 1990 AND 2000
 GROUP BY species;
 
+--animals belong to Melody Pond
+SELECT a.name
+FROM animals AS a
+JOIN owners AS o ON a.owner_id = o.id
+WHERE o.full_name = 'Melodie Pond';
+
 --
+SELECT animals.name
+FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
```

Comment:
- Hey, you've only added two queries, Please, kindly add the remaning queries as required from the project requirement.


![image](https://github.com/HossainAraf/vet-clinic-database/assets/59397159/116ad3c1-94d4-4279-9421-2708c006afbb)

Reply:
@zieeco Thanks for the feedback.